bandwidth-file-spec.txt

```diff
@@ -474,6 +563,36 @@ earliest_bandwidth=2018-04-16T15:13:26
 bw=380 error_circ=0 error_misc=0 error_stream=1 master_key_ed25519=YaqV4vbvPYKucElk297eVdNArDz9HtIwUoIeo0+cVIpQ nick=Test node_id=$68A483E05A2ABDCA6DA5A3EF8DB5177638A27F80 rtt=380 success=1 time=2018-05-08T16:13:26
 bw=189 error_circ=0 error_misc=0 error_stream=0 master_key_ed25519=a6a+dZadrQBtfSbmQkP7j2ardCmLnm5NJ4ZzkvDxbo0I nick=Test2 node_id=$96C15995F30895689291F455587BD94CA427B6FC rtt=378 success=1 time=2018-05-08T16:13:36
 
+A.3. Generated by sbws version 0.2.X
+
+1523911758
+version=1.1.0
+software=sbws
+software_version=0.1.0
```

Comment:
This is not sbws version 0.2.X either.

Reply:
I forgot to do that change in the code.
The 1st new version that can have all these output correct will be then 1.0.1, since the current version 1.0.0 don't have `version` changed either, i just forgot.